go.mod

```diff
@@ -78,7 +78,9 @@ require (
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/subosito/gotenv v1.3.0 // indirect
+	github.com/toorop/go-dkim v0.0.0-20201103131630-e1cd1a0a5208 // indirect
```

Comment:
Do you know why we need `go-dkim` in our go.mod here? I don't see it imported or used anywhere, so just wanted to double check that we still need this.

Reply:
This is likely an indirect dependency of GoSimpleMail to support certification of our SMTP server.

Reply:
Yep:
https://github.com/xhit/go-simple-mail/blob/bfa248936baba57cf414c64e6b10e2f5a1d01c34/go.mod#L3